services/postgres/schema.prisma

```diff
@@ -112,8 +115,21 @@ model RelayLedger {
     tenantId        String
     referenceId     String
     amount          Int
-    chainId         String
-    keyId           String // fed from redis(redis-pg sync service // tbd
+    product         Products        @relation(fields: [productId], references: [id])
+    productId       String
+    keyId           String // fed from redis(redis-pg sync service // tbd (@plor: should this be gone?)
```

Comment:
I don't think we need it. I think this was a hint to the service, but we aren't following this approach anymore.